lib/engine/game/g_1856/step/buy_sell_par_shares.rb

```diff
@@ -89,9 +89,11 @@ def can_gain?(entity, bundle, exchange: false)
 
             corporation = bundle.corporation
 
+            bundle_size = bundle.respond_to?(:shares) ? bundle.shares.sum(&:cert_size) : bundle.cert_size
```

Comment:
why wouldn't bundle have shares?

Reply:
If there is only one share available, instead of being a ShareBundle it is a Share at this point

Reply:
why? shouldn't everything always be a share bundle? all shares should be share.to_bundle, did you ovveride something?

Reply:
I added `puts 'base cb', entity, bundle` to `can_buy?` in `Engine::Step::BuySellParShares` and that shows sometimes being single shares being passed to `can_buy?` (and then to `can_gain?`)in unrelated games (tested in 1889)

![image](https://user-images.githubusercontent.com/2993555/123139171-ea9c7c80-d423-11eb-9522-53af733def08.png)


From searching the codebase it looks like the default behavior is:
Market/Treasury/IPO shares: Share
Par new Corporation: Bundle
Corporate Share Buy: Bundle

Reply:
ok, can we cahnge market treasure ipo to be bundle?

Reply:
I changed them to use `.to_bundle` and the tests passed locally and buying the 0.5 cert worked in 1856 after adjusting